### PR TITLE
Add signup redirect_uri param and pass through email.

### PR DESCRIPTION
**Pull request by sylmak**
Add signup redirect_uri param and pass through email.

Additional small fixes:
* send verification email on sign up
* fix email addresses

### CODE REVIEW

File: src/router.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ export const router = createRouter({
       path: '/action',
       component: HandleAction,
       props: (route: RouteLocation) => ({
+        continueUrl: route.query.continueUrl,
```
Comment (skeshive) [inline src/router.ts:42]: is this different from redirect_uri? if not, lets keep the name consistent

File: src/components/UpdateEmailCheckEmail.vue [hunk 2]
```diff
@@ -41,7 +41,7 @@ onBeforeMount(async () => {
   </p>
   <p>
     Didn't receive the email? Check your spam filter for an email from
-    account@climateiq.org.
+    noreply@climateiq.org.
```
Comment (skeshive) [inline src/components/UpdateEmailCheckEmail.vue:44]: noreply.climateiq.firebaseapp.com

File: src/components/SignUpCheckEmail.vue [hunk 3]
```diff
@@ -44,7 +50,7 @@ onBeforeUnmount(unsubscribeAuthListener);
   </p>
   <p>
     Didn't receive the email? Check your spam filter for an email from
-    account@climateiq.com.
+    noreply@climateiq.org.
```
Comment (skeshive) [inline src/components/SignUpCheckEmail.vue:53]: noreply.climateiq.firebaseapp.com

File: src/components/ResetPasswordCheckEmail.vue [hunk 4]
```diff
@@ -32,7 +32,7 @@ async function resendEmail() {
   </p>
   <p>
     Didn't receive the email? Check your spam filter for an email from
-    account@climateiq.org.
+    noreply@climateiq.org.
```
Comment (skeshive) [inline src/components/ResetPasswordCheckEmail.vue:35]: noreply.climateiq.firebaseapp.com

**Review by skeshive — APPROVED** [review]

File: src/components/UpdateEmailCheckEmail.vue (hunk 2 above)
Comment (sylmak) [reply src/components/UpdateEmailCheckEmail.vue:44]: It's supposed to be norely@climateiq.org - we're working on getting the email domain validated right now.

File: src/router.ts (hunk 1 above)
Comment (sylmak) [reply src/router.ts:42]: The query param must be named continueUrl: https://firebase.google.com/docs/auth/custom-email-handler#create_the_email_action_handler_page (see the params table)

**Pull request merged**